modyn/config/schema/pipeline/evaluation/strategy/between_two_triggers.py

```diff
@@ -0,0 +1,13 @@
+from typing import Literal
+
+from modyn.config.schema.base_model import ModynBaseModel
+from pydantic import Field
+
+
+class BetweenTwoTriggersEvalStrategyConfig(ModynBaseModel):
+    """This evaluation strategy will evaluate the model on the intervals between two consecutive triggers.
```

Comment:
probably nit, but you between two consecutive triggers, there is just a single interval, right? and you use the plural of interval because we have multiple triggers?